code/modules/power/floodlight.dm

```diff
@@ -155,6 +155,7 @@
 	. = ..()
 	RegisterSignal(src, COMSIG_OBJ_PAINTED, TYPE_PROC_REF(/obj/machinery/power/floodlight, on_color_change))  //update light color when color changes
 	register_context()
+	connect_to_network()
```

Comment:
```suggestion
	if(mapload)
		set_anchored(TRUE)
		connect_to_network()
```
This will do until we need to add a (broken?) floodlight that isn't anchored and working to a map.